umei-openapi.json

```diff
@@ -2519,6 +2540,17 @@
             "description": "Status of the order",
             "nullable": true
           },
+          "completionType": {
```

Comment:
CompletionType is meant to be a part of Orders, not Trades. I think if we want to have it in trades as well, we should probably do it in a different PR

Reply:
I wasn't part of the discussions on the UMEI project, so I'm not familiar with all the details of what was discussed. But, at least in NODES, not all fields are directly copied from the order to the trade. For example, price will be calculated based on the Buy and the Sell price, so it isn't necessary that it matches the price of the order. Status in the Trade is the status of the trade itself, which always start as Active, regardless of whether the Order is moving from Pending/Active to Completed/Filled/Killed or remaining as Active (in the case of a partial match). I think that makes sense, or otherwise it would read as if the trade itself was Killed?

Reply:
Indeed it make sense! You can keep the code as it is